app/templates/courses/teacher-class-view.jade

```diff
@@ -252,7 +252,7 @@ mixin allStudentsLevelProgressDot(progress, level, levelNumber)
       - labelText = ''
       - dotClass += ' practice'
 
-  if link && !state.get('readOnly')
+  if link
```

Comment:
Would this allow SA to click on just the project dot, or all the course progress dots?

Reply:
The SA should be able to view all the things a teacher can view, so we want to open this up completely to the SA.

Reply:
Yep sounds good, I got confused with the PR title and thought these changes are to just allow project gallery access to SAs. But makes sense since we are allowing student code access to SAs as well. Thanks.